cf-browser/src/main/java/org/eclipse/californium/tools/GUIDialog.java

```diff
@@ -96,10 +100,6 @@ private void secretToView() {
 			formatBase64.setSelected(true);
 			secret = secretConfig.base64;
 			format = "base64";
-		} else {
-			formatText.setSelected(true);
-			secret = "";
-			format = "text";
```

Comment:
how about also fixing the indentation in this method?

Reply:
Hm, I applied [Eclipse/Californium - eclipse-formatter-profile](https://github.com/eclipse/californium/blob/main/eclipse-formatter-profile.xml). So I'm not sure, what is wrong with the indentation?

Reply:
seems to be a display problem in the GitHub diff viewer. If you expand the code section above your change, then it looks like the indentation were wrong, doesn't it?

Reply:
Maybe, the main difference is, that beside a minimum (apply the formatter), I don't care too much about the format ;-). 
My firefox doesn't show something, which looks bad for me.
Do you more/less indention? We use tabs.

Reply:
never mind